packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -388,6 +388,34 @@ const ItemSelectedDisks = ({ disk, onDiskRemove }) => {
   )
 }
 
+const AdvancedSettings = decorate([
```

Comment:
IMO, `AdvancedSettings` should be in the `settings` section, like it is for backup creation/edition.